### DISCUSSION TITLE
ExportDataUnitsConfiguration example code?

### DISCUSSION THREAD

**Discussion by jwilson-lyonscg** [SalesforceCommerceCloud/sfcc-ci#520] [Team Posts]
I see minimal documentation about the ExportDataUnitsConfiguration in OCAPI. Is there an example somewhere of the format of this JSON object?

**Comment by mkimpelsf** [comment]
Here you go: https://documentation.b2c.commercecloud.salesforce.com/DOC1/index.jsp?topic=%2Fcom.demandware.dochelp%2FJobs%2FSiteExportSystemStep.html&resultof=%22siteexport%22%20

**Comment by jwilson-lyonscg** [comment]
Great thank you. Could not find this before. Appreciate it!

**Comment by JimmyKongoli** [comment]
Can we upload catalog images for the whole catalog  programmatically?

Thank you,
Jimmy

**Comment by djb2c** [comment]
Yes, absolutely.  Check out https://github.com/SalesforceCommerceCloud/sfcc-ci-assets-upload -- it provides a working example of how to upload storefront editorial assets using [SFCC-CI](https://github.com/SalesforceCommerceCloud/sfcc-ci).

**Comment by mamifofana** [comment]
Hello,
Yes !
